src/main/java/frc/robot/vision/data/AprilTagStandardDeviations.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.vision.data;
+
+public record AprilTagStandardDeviations(
```

Comment:
few notes: 
```suggestion
public record StandardDeviations3D(
```
- there's a 2D ver of that in the wpilib-poseestimator PR (#509), I think you should take a look
- A constructor from a double array would be nice. Currently in limelightSource (#501) there's a manual breakdown of that.